src/main/java/ch/njol/skript/util/BlockUtils.java

```diff
@@ -99,4 +101,21 @@ public static Location getLocation(final @Nullable Block b) {
 		return l;
 	}
 	
+	@Nullable
+	public static BlockData createBlockData(String dataString) {
+		String data = dataString.replace(";", ",");
```

Comment:
Maybe add a comment for the rationale behind the first replace

Reply:
Will do 👍